pkg/destroy/aws/aws.go

```diff
@@ -996,6 +998,30 @@ func deleteEC2NATGatewaysByVPC(ctx context.Context, client *ec2.EC2, vpc string,
 	return err
 }
 
+func deleteEC2PlacementGroup(ctx context.Context, client *ec2.EC2, id string, logger logrus.FieldLogger) error {
+	response, err := client.DescribePlacementGroupsWithContext(ctx, &ec2.DescribePlacementGroupsInput{
+		GroupIds: []*string{aws.String(id)},
+	})
+	if err != nil {
+		if err != nil {
```

Comment:
This is a duplicate check for `err != nil`.

Reply:
Good catch, have resolved and pushed over the top